src/components/implementation/no_interface/boot/booter.c

```diff
@@ -333,6 +333,7 @@ boot_find_cobjs(struct cobj_header *h, int n)
 		hs[i] = h = (struct cobj_header*)end;
```

Comment:
It is not clear that all of this will work with the booter (i.e. not the llbooter).  `booter.c` is general to all booters.  I suppose we can deal with this when we get to it.